src/main/java/node/Model/Peer.java

```diff
@@ -1,29 +1,16 @@
 package node.Model;
 
-import java.net.InetAddress;
-import java.net.UnknownHostException;
-
 public class Peer {
-    private InetAddress ipAddress;
+    private String ipAddress;
```

Comment:
please contact with Tobys, he as adress used InetAddress class and clarify way to do it the same way

Reply:
Peer class is used to parse json to object so this has to be ipAddress in this case has to be String.